### ISSUE TITLE
Api change?

### ISSUE THREAD

**Issue by jusplainmike** [noelportugal/alexa-reminders#3]
Is this still working for anyone? Trying the basic example doesn't do anything. Maybe the API has changed? There is no error logging so dunno whats going on.

**Comment by nerrixde** [comment]
same

**Comment by Yuhanlolo** [comment]
@jusplainmike Yes I've been using this for a while and it still works for me. You may want to check if you've successfully login to Alexa web app before requesting a reminder. I would suggest login first and store the config somewhere and reuse it later.

**Comment by jusplainmike** [comment]
Thanks buddy I'll take another crack at it tonight.
